admin/workspace-management/cvms/index.md

```diff
@@ -28,6 +28,20 @@ deployment option.
 > [Google](../../../setup/kubernetes/google.md). Azure-hosted clusters will meet
 > these requirements as long as you use Kubernetes version 1.18+.
 
+### HostPath mounts
+
+The host paths required for CVM functionality depend on whether **Caching** is
```

Comment:
Is it worth mentioning that Sysbox requires kernel source headers in order to run Kubernetes (e.g. kind) inside the sysbox container?

Also, here's docs for the sysbox mounts: https://github.com/nestybox/sysbox/blob/master/docs/user-guide/design.md#mounts

Not sure how much detail we want, I think the existing description is fine too